tests/e2e/logging/test_openshift-logging.py

```diff
@@ -6,27 +7,31 @@
 import logging
 
 from tests import helpers
+from ocs_ci.ocs.ocp import OCP
```

Comment:
Can you do alphabetical order of the imports please? This follow PEP8 standards.